yamscripts.yml

```diff
@@ -59,7 +59,7 @@ scripts:
     pre: =>prebuildmc
     copy: =>buildmc:copy -- -w
     webpack: =>buildmc:webpack -- -w
-    css: =>buildmc:css && =>buildmc:css -- -w
+    css: =>buildmc:css && nodemon --ext scss -x "npm run buildmc:css" --watch system-addon/content-src -q
```

Comment:
Looks like we can silent the rest of npm error with a `-s`, so `-x "npm run buildmc:css -s"`